templates/chrony.conf.epp

```diff
@@ -1,12 +1,42 @@
 # NTP servers
+<% if $chrony::servers.is_a(Hash) { -%>
+<% $chrony::servers.keys.sort.each |$server| { -%>
+<% if $chrony::servers[$server] and !$chrony::servers[$server].empty { -%>
```

Comment:
There is a readability trick you can use to make it a bit more readable.

```suggestion
<%   $chrony::servers.keys.sort.each |$server| { -%>
<%     if $chrony::servers[$server] and !$chrony::servers[$server].empty { -%>
```

Reply:
Done.